app/models/concerns/twitch_user.rb

```diff
@@ -12,7 +12,7 @@ def follows
     end
 
     def twitch_sync!
-      body = Twitch::User.get(name)
+      body = Twitch::User.get(self)
```

Comment:
For code in `lib/` it's generally good to keep inputs super simple (ids rather than whole ActiveRecord objects) so that the library stays hyperfocused on abstracting the communication part and doesn't accidentally take on other responsibilities.